pages/common/tsv-filter.md

```diff
@@ -0,0 +1,36 @@
+# tsv-filter
+
+> Filter lines of a TSV file by running tests against individual fields.
+> More information: <https://github.com/eBay/tsv-utils#tsv-filter>.
+
+- Output all lines where column with field_name is numerically equal to value:
```

Comment:
`output` is a verb hard to translate to other languages.
```suggestion
- Print the lines where a specific column is numerically equal to a given number:
```
I think `number` is better to find the placeholder in the command.